.pre-commit-config.yaml

```diff
@@ -8,16 +8,37 @@ repos:
   rev: v4.4.0
   hooks:
   - id: check-ast
-  - id: trailing-whitespace
-  - id: end-of-file-fixer
   - id: check-merge-conflict
     exclude: .*\.rst
-  - id: check-symlinks
   - id: check-case-conflict
+  - id: name-tests-test
+    args: [--pytest-test-first]
+    exclude: plasmapy/tests/_helpers/.*
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
   - id: check-json
   - id: check-toml
   - id: check-yaml
 
+- repo: https://github.com/sirosen/texthooks
+  rev: 0.5.0
+  hooks:
+  - id: fix-smartquotes
+  - id: fix-spaces
+  - id: fix-ligatures
+  - id: forbid-bidi-controls
+
+- repo: https://github.com/Lucas-C/pre-commit-hooks
+  rev: v1.5.4
+  hooks:
+  - id: remove-crlf
```

Comment:
This makes sure that the line ending is consistently LF (Unix style) rather than CRLF (Windows style).  See also #2331.